### PR TITLE
Cockpit 275

### PR DESCRIPTION
**Pull request by jelly**
Preview available here https://jelly.github.io/cockpit-project.github.io/blog/cockpit-275.html

### CODE REVIEW

**Comment by marusak** [comment]
pushed tag, can you please re-run without the preview mode?

File: _posts/2022-08-24-cockpit-275.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+![screenshot of add read/write usage per disk]({{ site.baseurl }}/images/275-metrics-add-read-write-usage-per-disk-3.png)
+
+## Shell: Support for alternatives to sudo
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/17536>
+{:.note}
+
+Cockpit always had the ability to run arbitrary programs to gain administrative access, by adding them to a package manifest.  However, the UI would always use "sudo". Now the UI lets you select which program to use when multiple are defined in the manifests.
+
+By default, Cockpit will still only use "sudo", so unless you actually add more programs via a manifest, there will be no visible change in the UI.
```
Comment (allisonkarlitskaya) [inline _posts/2022-08-24-cockpit-275.md:40]: Do we want to link to the youtube video for this?

File: _posts/2022-08-24-cockpit-275.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+## Image history
```
Comment (allisonkarlitskaya) [inline _posts/2022-08-24-cockpit-275.md:43]: ## cockpit-podman: Image history

File: _posts/2022-08-24-cockpit-275.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1074>
+{:.note}
+
+The image detail view now has a new tab which can show the image history similar to `podman image history`.
+
+![screenshot of ## image history]({{ site.baseurl }}/images/275-image-history.png)
+
+## Restart policy available for users with lingering enabled
```
Comment (allisonkarlitskaya) [inline _posts/2022-08-24-cockpit-275.md:52]: ditto, as above

File: _posts/2022-08-24-cockpit-275.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+By default, Cockpit will still only use "sudo", so unless you actually add more programs via a manifest, there will be no visible change in the UI.
+
+
+## Image history
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1074>
+{:.note}
+
+The image detail view now has a new tab which can show the image history similar to `podman image history`.
```
Comment (allisonkarlitskaya) [inline _posts/2022-08-24-cockpit-275.md:48]:
```suggestion
The image detail view now has a new tab which shows the image history similar to `podman image history`.
```

File: _posts/2022-08-24-cockpit-275.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+
+Although lingering is not recommended, cockpit-podman now supports setting a restart policy when a user has enabled lingering.
+
+## Display ports, volumes and environment variables
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit-podman/pull/1017>
+{:.note}
+
+Published ports, mounted volumes and environment variables are now visible in container integration tab.
+By default only variables set by user are shown and after clicking `Show more` all variables are visible.
```
Comment (allisonkarlitskaya) [inline _posts/2022-08-24-cockpit-275.md:65]:
```suggestion
By default only variables set by user are shown.  After clicking `Show more` all variables are visible.
```

File: _posts/2022-08-24-cockpit-275.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+On phone:
+![screenshot of add read/write usage per disk]({{ site.baseurl }}/images/275-metrics-add-read-write-usage-per-disk-3.png)
+
+## Shell: Support for alternatives to sudo
+
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/17536>
+{:.note}
+
+Cockpit always had the ability to run arbitrary programs to gain administrative access, by adding them to a package manifest.  However, the UI would always use "sudo". Now the UI lets you select which program to use when multiple are defined in the manifests.
```
Comment (allisonkarlitskaya) [inline _posts/2022-08-24-cockpit-275.md:38]:
```suggestion
Cockpit has always had the theoretical ability to run arbitrary programs to gain administrative access, by adding them to a package manifest.  However, the UI would always use "sudo". Now, if multiple programs are defined, the UI will prompt for which one to use.
```

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

File: _posts/2022-08-24-cockpit-275.md (hunk 2 above)
Comment (jelly) [reply _posts/2022-08-24-cockpit-275.md]: We use `Images` & `Podman` in older release notes so I kept it consistent. Maybe _scripts/generate-release-notes should prefix.

File: _posts/2022-08-24-cockpit-275.md (hunk 1 above)
Comment (jelly) [reply _posts/2022-08-24-cockpit-275.md]: Do we usually link to videos?

File: _posts/2022-08-25-cockpit-275.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 275 and cockpit-podman 52:
+
+
+## Shell: Support for alternatives to sudo
+
+Cockpit has always the theoretical ability to run arbitrary programs to gain administrative access, by adding them to a package manifest.  However, the UI would always use "sudo". Now, if multiple programs are defined, the UI will prompt for which one to use.
```
Comment (mvollmer) [inline _posts/2022-08-25-cockpit-275.md:19]: "Cockpit always had the theoretical ability"?

File: _posts/2022-08-24-cockpit-275.md (hunk 1 above)
Comment (mvollmer) [reply _posts/2022-08-24-cockpit-275.md]: The video is not that great... We could point people to the new documentation for the "label" property of a bridge.

File: _posts/2022-08-25-cockpit-275.md (hunk 7 above)
Comment (allisonkarlitskaya) [reply _posts/2022-08-25-cockpit-275.md:19]: *has always had, i'd say

Comment (allisonkarlitskaya) [reply _posts/2022-08-25-cockpit-275.md:19]: (don't ask me to explain this.  english is weird.)

Comment (jelly) [reply _posts/2022-08-25-cockpit-275.md]: Applied!

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**